notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>Should this output be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>